doc/yast-storage-requirements.md

```diff
@@ -0,0 +1,293 @@
+# YaST Storage Features and Requirements #
```

Comment:
All along the document it's not so clear what is an existing feature and what is a (new?) requirement.